docs/src/app/components/pages/components/GridList/Page.jsx

```diff
@@ -11,17 +11,31 @@ import GridListExampleSimple from './ExampleSimple';
 import gridListExampleComplexCode from '!raw!./ExampleComplex';
 import GridListExampleComplex from './ExampleComplex';
 
+const descriptions = {
+  simple: 'A simple example of a scrollable `GridList`.',
+  complex: 'This example demonstrates "featured" tiles, using the `rows` and `cols` props to adjust the size of the ' +
+  'tile. The tiles have a customised title, positioned at the top and with a custom gradient `titleBackground`.',
+};
+
 const GridListPage = () => (
   <div>
     <MarkdownElement text={gridListReadmeText} />
-    <CodeExample code={gridListExampleSimpleCode}>
+    <CodeExample
+      title="Simple Grid List"
+      description={descriptions.simple}
+      code={gridListExampleSimpleCode}
+    >
       <GridListExampleSimple />
     </CodeExample>
-    <CodeExample code={gridListExampleComplexCode} >
+    <CodeExample
+      title="Complex Grid List"
+      description={descriptions.complex}
+      code={gridListExampleComplexCode}
+    >
       <GridListExampleComplex />
     </CodeExample>
-    <PropTypeDescription code={gridListCode}/>
-    <PropTypeDescription code={gridTileCode}/>
+    <PropTypeDescription header="### Grid List Properties" code={gridListCode}/>
+    <PropTypeDescription header="### Grid Tile Properties" code={gridTileCode}/>
```

Comment:
To clarify my point of view, I think that:
- For feature with only one component `Properties` is enough. We are already in the context of this specific component.
- For feature with more than one component, we need to specify the component name the properties are linked to. Hence, I think that the component.displayName will be better.
- For examples, I don't think that `Controlled example` is better for the same reason that we should use:
```js
    /**
     * The css class name of the root element.
     */
    className: React.PropTypes.string,
``` 
and not:
```js
    /**
     * Applied to the app bar's root element.
     */
    className: React.PropTypes.string,
```